.github/workflows/python-ci.yml

```diff
@@ -7,8 +7,8 @@ jobs:
       matrix:
         include:
           - spark-version: 3.5.4
-            scala-version: 2.12.18
-            python-version: 3.9.19
+            scala-version: 2.12.20
```

Comment:
@bjornjorgensen I backed a lot of stuff out. Now this is the only unrelated commit to building a `graphframes` package.